### PR TITLE
added tips about performance schema, global variables, and Aurora multi-AZ considerations

### PR DESCRIPTION
**Pull request by danhermann**

### CODE REVIEW

**Comment by kimschmidtsbrain** [comment]
I have 25 videos on YouTube on Aurora here: https://youtu.be/Cnz6mSzca1Y?list=PL2EgRNBrbhC4lbTLy9cZnHG2JvEd8wZCr Almost done with section 2.0 on Architecture.

Kim

**Comment by danhermann** [comment]
I'm looking forward to seeing those. I still have my heaviest workloads on MySQL rather than Aurora, so anything I can learn in advance of migrating them will be helpful.

**Comment by kimschmidtsbrain** [comment]
Migration is planned for the 5th Section, have 1 more video to do in 2nd Section, then Configuration & Management, then Demos, then Migration, then BI/Analytics & Visualization - 6 total sections. Wanted them done before re:Invent but doesn't look like that will happen. However Performance, Global Variables, Multi-AZ is covered quite deeply in Video Section 2.0 in architecture, but some things are covered in the 1.0 Overview section. 

How soon do you need this info? Anything else? 

The Architecture section dives deep....perhaps start there & go back to relevant Overview sections if needed.

If you have certain topics other than Migration, let me know.

For Migration, I have planned sections on Overview, from MySQL Snapshot (sounds good for you), from MySQL External to Amazon RDS, from a DB that's not MySQL compatible.

Kim

**Comment by danhermann** [comment]
Thanks, Kim. It's quite likely that I won't have a change to do the biggest migrations until January. I'm currently working on an e-commerce app and they understandably want to minimize any big changes right before the holiday shopping season.

File: README.md [hunk 1]
```diff
@@ -1053,13 +1053,15 @@ RDS MySQL and MariaDB
 
 -	MySQL RDS allows access to [binary logs](http://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_LogAccess.Concepts.MySQL.html#USER_LogAccess.MySQL.BinaryFormat).
 -	Multi-AZ instances of MySQL transparently replicate data across AZs using DRBD. Automated backups of multi-AZ instances [run off the backup instance](https://www.percona.com/live/mysql-conference-2014/sessions/rds-mysql-tips-patterns-and-common-pitfalls) to reduce latency spikes on the primary.  
+-	🔸**Performance Schema** While Performance Schema is enabled by default in MySQL 5.6.6 and later, it is disabled by default in all versions of RDS. If you wish to enable Performance Schema, a reboot of the RDS instance will be required.
```
Comment (jlevy) [inline README.md]: Minor: Add colon and link:

>🔸**Performance Schema:** While [Performance Schema](http://dev.mysql.com/doc/refman/en/performance-schema.html) is enabled by default in MySQL 5.6.6 and later, it is disabled by default in all versions of RDS. If you wish to enable Performance Schema, a reboot of the RDS instance will be required.

File: README.md [hunk 2]
```diff
@@ -1053,13 +1053,15 @@ RDS MySQL and MariaDB
...
 
 ### RDS MySQL and MariaDB Gotchas and Limitations
 
 -	🔸**No SUPER privileges.** RDS provides some [stored procedures](http://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Appendix.MySQL.SQLRef.html) to perform some tasks that require SUPER privileges such as starting or stopping replication.
 -	🔸You can replicate to non-RDS instances of MySQL, but [replication to these instances will break during AZ failovers](https://www.percona.com/live/mysql-conference-2014/sessions/rds-mysql-tips-patterns-and-common-pitfalls).
 -	🔸There is no ability to manually CHANGE MASTER on replicas, so they must all be rebuilt after a failover of the master.
+-	🔸Most global options are exposed only via parameter groups. Some variables that were introduced in later MySQL dot releases such as [avoid_temporal_upgrade](https://dev.mysql.com/doc/refman/5.6/en/server-system-variables.html#sysvar_avoid_temporal_upgrade) in MySQL 5.6.24 are not made available in RDS's 5.6.x parameter group and making use of them requires an upgrade to MySQL 5.7.x.
```
Comment (jlevy) [inline README.md]: Nit: boldface the key concept and add another link. And I think it's "DB parameter groups" to be consistent with AWS, right?

> -	🔸Most global options are exposed only via [**DB parameter groups**](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_WorkingWithParamGroups.html). Some variables that were introduced in later MySQL dot releases such as [avoid_temporal_upgrade](https://dev.mysql.com/doc/refman/5.6/en/server-system-variables.html#sysvar_avoid_temporal_upgrade) in MySQL 5.6.24 are not made available in RDS’s 5.6.x parameter group and making use of them requires an upgrade to MySQL 5.7.x.

**Comment by jlevy** [comment]
Thanks @danhermann for the great additions!

**Comment by QuinnyPig** [comment]
@danhermann If you can do a rebase, let's get this merged.

**Comment by jlevy** [comment]
👍 Awesome. Thank you @danhermann and @QuinnyPig , good to finally get this in!

**Pull request merged**